src/content/blog/2024-11-06-supporting-emergency-dispatch-through-software-migration.mdx

```diff
@@ -0,0 +1,34 @@
+date: '2024-11-06T18:19:40.894Z'
+category: Collaboration
+cover_image: /src/images/pillar-blog/default-social-card.png
+cover_image_alt: ugrc social card
+---
+
+{/* remove if not using images
+import { Image } from 'astro:assets'; */}
+
+{/* remove if not using an image
+import myImage from '@images/blog/2024-11-06-supporting-emergency-dispatch-through-software-migration/image.png'; */}
+
+Have you seen any geese flying overhead lately? As the weather turns colder and the first snowflakes fall, many migratory bird species are making their way to warmer climates. Geese, with their iconic V-shaped flying formation, are an excellent example of an efficient migration. See, they don't just fly in that pattern for fun or to show off. The V-shaped pattern actually helps to decrease air resistance over the birds' wings, making the flight much more efficient than flying solo. This helps them save precious energy and caloric resources as they make their long journey south. Geese give us a good example of how all migrations should be: efficient, time-saving, and resource-conserving. So, when we talk about software migrations, we ought to take a page out of the avian book.
+
+Software migrations generally don't take us to sunnier places, at least not intentionally (if you've ever migrated to a new software and found yourself capable of teleportation, please let us know. We'd like to try it!). But migrating to new software can increase efficiency, streamline workflows, and open up new possibilities for everyone involved. Having the best possible software is especially important in emergency management, which is why UGRC has partnered with Utah's Public Safety Answering Point (PSAP) call centers to upgrade the software they use to handle 911 calls across the state. 
+
+Every time you dial 911 (whether your emergency involves geese or not), your call is routed to a specific PSAP communications center depending on your location. These PSAPs are responsible for ensuring emergency responders get to where they need to be as fast as possible. In the past, this was accomplished without any GIS involved. Each PSAP maintained their own records of addresses and locations in their dispatch area and used these to tell first responders where to go. This system relied on the knowledge the PSAP and local responders had of that area.
+
+Today's 911 systems include more than just matching an address heard over the phone to one in a table. Through Next Generation 911 (NG911), dispatchers can access exact point locations on a real-time map. This is accomplished through geovalidation, a process by which addresses and real-world locations are verified to ensure spatial accuracy. UGRC already maintains several data layers that are involved in this process, including Address Points, Roads, and PSAP boundaries. Backed by these statewide datasets and GIS-enabled systems, dispatchers can greatly speed up the time it takes to handle a 911 call and respond to an emergency situation.
```

Comment:
**thought**: It would be great to link out to the data pages for these data sets.